olp-cpp-sdk-dataservice-read/src/repositories/PartitionsRepository.cpp

```diff
@@ -359,13 +359,12 @@ model::Partition PartitionsRepository::PartitionFromSubQuad(
   ret.SetCompressedDataSize(sub_quad.GetCompressedDataSize());
   return ret;
 }
-
-PartitionResponse PartitionsRepository::GetAggregatedTile(
+PartitionResponse PartitionsRepository::QueryQuadTreeIndexAndGetTile(
```

Comment:
Because you merged it, I would recommend to add the aggregated flag to all logs so that we know from where the call came.